sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalProcessStreamConsumer.java

```diff
@@ -35,33 +36,52 @@
 @BatchSide
 @ScannerSide
 @SonarLintSide
```

Comment:
Do you need it to be SonarLintSide? It will be started for every analysis (since it is Startable now)

Reply:
Indeed, not needed anymore. I removed `@SonarLintSide` annotation